bgpd/bgp_labelpool.c

```diff
@@ -1545,3 +1572,71 @@ void bgp_lp_vty_init(void)
 	install_element(ENABLE_NODE, &clear_labelpool_perf_test_cmd);
 #endif /* BGP_LABELPOOL_ENABLE_TESTS */
 }
+
+DEFINE_MTYPE_STATIC(BGPD, LABEL_PER_NEXTHOP_CACHE,
+		    "BGP Label Per Nexthop entry");
+
+/* The nexthops values are compared to
+ * find in the tree the appropriate cache entry
+ */
+int bgp_label_per_nexthop_cache_compare(
```

Comment:
Let's use _cmp(), as in most of the other places.

Reply:
done